### ISSUE TITLE
Training issue: Ran out of patience. Stopping Training

### ISSUE THREAD

**Issue by NatCusack** [benbogin/spider-schema-gnn#5]
For some reason when I am training the model it is stopped with the error:
 `2019-07-02 13:16:56,613 - INFO - allennlp.training.trainer - Ran out of patience.  Stopping training.
2019-07-02 13:16:56,614 - INFO - allennlp.training.checkpointer - loading best weights
2019-07-02 13:16:56,635 - INFO - allennlp.models.archival - archiving weights and vocabulary to experiments/name_of_experiement/model.tar.gz
2019-07-02 13:16:57,984 - INFO - allennlp.common.util - Metrics: {
  "best_epoch": 12,
  "best_validation__match/exact_match": 0.327852998065764,
  "best_validation_sql_match": 0.43036750483558994,
  "best_validation__others/action_similarity": 0.5266099921420543,
  "best_validation__match/match_single": 0.552158273381295,
  "best_validation__match/match_hard": 0.28870292887029286,
  "best_validation_beam_hit": 0.5674081237911025,
  "best_validation_loss": 7.97007417678833,
  "peak_cpu_memory_MB": 5171.892
}
`
What does this mean?

**Comment by benbogin** [comment]
It means that the validation accuracy was not improved for X epochs (determined by the patience value) so training was stopped. The easiest way to keep training longer would be to re-run the training without patience (simply increase the "patience" value in the settings file, or set it to 0 to disable it).

**Comment by NatCusack** [comment]
Great! Thanks for your help!
